spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +454,32 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker") {
+    declare local var.variant STRING;
+    set var.variant = req.http.GOVUK-ABTest-StartABusinessSegment;
+
+    declare local var.sab_expiry TIME;
+    if (var.variant == "C") {
+      # If they have opted out of the banner or have visited the SaB checker
+      # Expire in 26 weeks (approx 6 months)
+      set var.sab_expiry = time.add(now, std.integer2time(15724800));
+    } else {
+      # Otherwise, expire in half an hour, approx this session
+      set var.sab_expiry = time.add(now, std.integer2time(1800));
+    }
+
+    if (req.http.GOVUK-ABTest-IsStartABusinessPage == "true" && var.variant == "A") {
+      # If the user is in variant A and has seen a SaB page then we need to move them to B
+      # so they see the banner on the next SaB page they hit.
+      set var.variant = "B";
+    }
+
+    if (std.strlen(var.variant) > 0) {
```

Comment:
Will `var.variant` always be populated when we're running this test? If so, the check to ensure that the test is active in an earlier comment should be enough to be able to remove this check, because you're always setting the variant back in the block from line 162?

Reply:
No, it's not guaranteed to be populated so we do want to keep the check - good question though